modules/ROOT/attachments/admin-api.yaml

```diff
@@ -584,6 +584,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/partitions_local_summary'
+  /partitions/majority_lost:
+    get: 
+      tags: 
+      - Partitions
+      summary: Get partitions with lost majority
+      description: List of partitions with lost majority given an input set of dead nodes.
+      operationId: majority_lost
+      responses: 
+        200:
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ntp_with_majority_loss'
+      parameters:
+        - name: dead_nodes
+          in: query
+          required: true
+          schema:
+            type: string   
```

Comment:
Need to double check this, spec indicates a string query parameter, but the force recover operation which also requires an input list of nodes accepts an array in the [request body](https://github.com/redpanda-data/redpanda/blob/e9b93b9cd397862145e33a7e193b118c85495c57/src/v/redpanda/admin/partition.cc#L1019)

Reply:
> /partitions/majority_lost?dead_nodes=1,3,5

yes you are correct, thats how the URL  looks. A GET with a request_body is not considered a standard practice in the implementations, so I had to use a query parameter.
https://github.com/redpanda-data/redpanda/pull/13943#issuecomment-1814103858